package.json

```diff
@@ -11,14 +11,19 @@
   },
   "scripts": {
     "build": "npm run build:css",
+    "watch": "npm run watch:css",
+    "build-dev": "npm run build-dev:css",
+    "watch-dev": "npm run watch-dev:css",
 
     "build:css": "npm run build:less",
     "watch:css": "npm run watch:less",
+    "build-dev:css": "npm run build-dev:less",
+    "watch-dev:css": "npm run watch-dev:less",
 
-    "build:less": "grunt less",
-    "lessdev": "grunt lessdev",
-    "watch:less": "grunt watch:less",
-    "watch:lessdev": "grunt watch:lessdev",
+    "build:less": "grunt less && npm run lessToSass",
+    "watch:less": "grunt watch:less && npm run watch:lessToSass",
```

Comment:
I don't think this works -- don't you need to use something like concurrently to get multiple watches working simultaneously? I did a test with this and when I changed a file, the code compiled, but the lessToSass step did not occur.

Reply:
That's a good point. Maybe I was hoping that the lessToSass would happen after the watch but that doesn't make sense. I'll skip the lessToSass here.